pkg/restore/restore.go

```diff
@@ -865,11 +866,18 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 
 			if shouldRenamePV {
 				// give obj a new name, and record the mapping between the old and new names
-				oldName := obj.GetName()
-				newName, err := ctx.pvRenamer(oldName)
-				if err != nil {
-					addToResult(&errs, namespace, errors.Wrapf(err, "error renaming PV"))
-					return warnings, errs
+				var newName string
```

Comment:
The `newName` var makes sense in the context of ln 873, but not on ln 880, since the same name is being used. I propose this var be named just `name`, to avoid being misleading in the case where the name will not be a new name.